apps/www/_blog/2024-07-11-hardening-supabase.mdx

```diff
@@ -0,0 +1,169 @@
+---
+title: 'Supabase Security Suite'
+description: 'Learn how to use range columns in Postgres to simplify time-based queries and add constraints to prevent overlaps.'
+author: paul_copplestone
+image: security-suite/hardening-og.png
+thumb: security-suite/hardening-thumb.png
+categories:
+  - product
+tags:
+  - security
+date: '2024-07-11'
+toc_depth: 3
+---
+
+Over the past 3 months, the team has been focused on **security**, **performance**, and **stability**. This is always part of our mandate, but for this period we have been _exclusively_ focused on this task.
+
+And we haven't been alone with security. [@xyzeva](https://x.com/xyz3va/) (Eva), a rising star in the world of security, has been instrumental in the past 3 months - everything from discovering misconfigured projects to collaborating on fixes and features.
+
+This post outlines the key initiatives that we have collaborated on and a few more in the pipeline.
+
+## Learning from our peers
+
+Eva and her colleagues are the authors of [Firewreck](https://env.fail/posts/firewreck-1/), which exposed the misconfiguration of Firebase instances. This [HN comment](https://news.ycombinator.com/item?id=39743225) highlights the key challenges for Firebase (paraphrased):
+
+> .. First, security rules as implemented by Firebase are still a novel concept.
+
+.. Second, without the security of obscurity created by random in-house implementations of backends, scanning en masse becomes easier.
```

Comment:
I think this line and the following sentence are both meant to be quoted with > along with the above line.